scripts/build/bundler.js

```diff
@@ -205,11 +211,25 @@ function getRollupOutputOptions(bundle) {
   if (bundle.target === "node") {
     options.format = "cjs";
   } else if (bundle.target === "universal") {
-    options.format = "umd";
     options.name =
       bundle.type === "plugin" ? `prettierPlugins.${bundle.name}` : bundle.name;
+
+    if (!bundle.format && bundle.bundler !== "webpack") {
+      return [
+        {
+          ...options,
+          format: "umd",
+        },
+        {
+          ...options,
+          format: "esm",
+          file: `dist/esm/${bundle.output.replace(".js", ".mjs")}`,
```

Comment:
I’d prefer if this was done using `dist/esm/package.json` with content <code>{ "[type][package-json-type-field]": "module" }</code> and:

```suggestion
          file: `dist/esm/${bundle.output}`,
```

[package-json-type-field]: https://nodejs.org/api/esm.html#esm_package_json_type_field

Reply:
We are making if work in browser, not node.js.

Reply:
And browsers prefer the `.js` file extension.

Reply:
Even for npm package, I prefer use `.mjs` extension.

Reply:
> And browsers prefer the `.js` file extension.

Who says that?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#Aside_%E2%80%94_.mjs_versus_.js